applications/ConvectionDiffusionApplication/custom_elements/qs_convection_diffusion_explicit.cpp

```diff
@@ -231,6 +231,8 @@ void QSConvectionDiffusionExplicit<2,3>::CalculateLumpedMassVector(
     for (IndexType i = 0; i < local_size; ++i) {
         rLumpedMassVector(i) = one_third;
     }
+    // Assumption all the Gauss points have the same weight, so we multiply by the volume
+    rLumpedMassVector *= GetGeometry().Volume();
```

Comment:
Why not multiplying the `one_third` by the volume. Internally, the  `*=` is multiplying each component of the vector. Considering that this is an explicit strategy any optimization becomes relevant.

Reply:
You are right